androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/PermissionRequestActivity.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.androidbrowserhelper.trusted;
+
+import android.app.Activity;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.os.Message;
+import android.os.Messenger;
+import android.os.RemoteException;
+
+import android.util.Log;
+import androidx.core.app.ActivityCompat;
+
+/**
+ * This is a simple transparent activity that will bring up the permission prompt. On either approve
+ * or disapprove, this will send the result via the {@link Messenger} provided with the intent, and
+ * then finish.
+ */
+public class PermissionRequestActivity extends Activity {
```

Comment:
What does this Activity look like? We're not setting the style anywhere?

Reply:
Let's talk about this next time you're in office.